backend/src/main/java/net/wodie/backend/model/BtoItem.java

```diff
@@ -0,0 +1,28 @@
+package net.wodie.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Document(collection= "btoItems")
+public class BtoItem {
+
+    @Id
+    private String id;
+    private String displayId;
+    private String category;
+    private String titel1;
+    private String titel2;
+    private String description;
+    private String status;
+    private String actionOwner;
+    private String[] actionMembers;
```

Comment:
`String[]` funktioniert zwar, aber `List<String>` (oder dergleichen) ist in der Regel einfacher im Code zu handhaben.

Reply:
Wurde geändert wie vorgeschlagen